src/main/java/ru/l3r8y/rule/WorkerCheck.java

```diff
@@ -53,7 +48,12 @@ public final class ErNamedClass implements Rule {
     public Collection<Complaint> complaints() {
         return new ConditionRule(
             this::isEndsWithEr,
-            new WrongClassNaming(this.name, "class ends with '-er' suffix")
+            new WrongClassNaming(
+                this.name,
+                // @checkstyle StringLiteralsConcatenationCheck (4 lines).
+                "class ends with '-er' suffix, it's prohibited,"
+                + " read: https://www.yegor256.com/2015/03/09/objects-end-with-er.html"
```

Comment:
not sure about `+` concatenation here

Reply:
i think you can add some decorator for `Rule` like `LinkedRule` with link to resource, which will be added to warning message

Reply:
go ahead